api/holiday_acres_api/migrations/0003_remove_paddock_last_maintained.py

```diff
@@ -0,0 +1,10 @@
+# Generated by Django 3.2.7 on 2021-10-04 01:35
```

Comment:
Can you condense these down to a single migration?

Reply:
I believe I fixed the issue with commit `e4c86e8`, but I thought that I hadn't, so I kept messing with it. 
Now I don't know how to remove the following 3 commits(`cb3a691, 683da18, dc6e027`). 
They don't actually change anything that I can tell, I just wanted to remove them to clean things up.